### ISSUE TITLE
Some example .py files missing from pip-installed TensorFlow source tree

### ISSUE THREAD

**Issue by toph-allen** [tensorflow/tensorflow#6196]
I'm working through TensorFlow's [the RNN tutorial](https://www.tensorflow.org/versions/master/tutorials/recurrent/index.html). Navigating to the `tensorflow/models/rnn/ptb` directory, I should see `reader.py` and `ptb_word_lm.py`, but I only see `reader.py` (alongside `__init__.py` etc).

I've tried both the install using `https://storage.googleapis.com/tensorflow/mac/cpu/tensorflow-0.12.0rc0-py3-none-any.whl`, and the default pip installation; they don't differ in this regard.

I wonder if this is related to #5014, which also deals with missing files. Everything seems to _work_ correctly, as far as I can tell — it's just that some of these examples are missing from the source tree.

### Environment info

- Operating System: macOS 10.12.1
- I'm using Homebrew'd Python 3.5 in a virtualenv.
- The output from `python -c "import tensorflow; print(tensorflow.__version__)"`: `0.12.0-rc0`

**Comment by michaelisard** [comment]
@petewarden is this deliberate?

**Comment by petewarden** [comment]
No, I believe this is a problem that has been introduced by moving the PTB examples over to the tensorflow models repo. If you follow the tutorial, it points you to tensorflow/models/rnn/ptb, but the code is now in https://github.com/tensorflow/models/tree/master/tutorials/rnn

@nealwu can you take a look, since you were involved in the move?

**Comment by nealwu** [comment]
Thanks for reporting this. I've already fixed this tutorial to clarify where to find those model files after the move, so it should be visible the next time we push out an update to the TensorFlow site.

In the meantime, use this folder as a reference: https://github.com/tensorflow/models/tree/master/tutorials.

**Comment by toph-allen** [comment]
Thanks! That's helpful.

**Comment by tedsandler** [comment]
@nealwu line 133 of "ptb_word_lm.py" has a comment that reads:

     # The alternative version of the code below is:
     #
     # inputs = tf.unstack(inputs, num=num_steps, axis=1)
     # outputs, state = tf.nn.rnn(cell, inputs,
     #                            initial_state=self._initial_state)

https://github.com/tensorflow/models/blob/master/tutorials/rnn/ptb/ptb_word_lm.py#L133

However, it is unclear from the comment which code below needs to be replaced. Can you speak to this?

Thanks
Ted

**Comment by nealwu** [comment]
@tedsandler It replaces the 7-line chunk of code right below it that generates `outputs` and `state`.

Also, this is an issue someone else opened and isn't the best place to ask such a question. In the future if you believe there is an issue with the code you can open a new issue, or if you just have a question like this you should ask on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow).
